src/prog_models/models/uav_model/uav_model.py

```diff
@@ -0,0 +1,387 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+import numpy as np
+from warnings import warn
+
+from prog_models.prognostics_model import PrognosticsModel
+from prog_models.models.uav_model.vehicles.aero import aerodynamics as aero
+from prog_models.models.uav_model.vehicles import vehicles
+from prog_models.utils.traj_gen import geometry as geom
+
+# Small Rotorcraft Model
+# ======================
+class SmallRotorcraft(PrognosticsModel):
+
+    """
```

Comment:
```suggestion
    r"""
```
Force it to be a raw string. This prevents the \t from theta as being interpreted as a tab.